buildkite/provider.go

```diff
@@ -83,6 +83,26 @@ func (tf *terraformProvider) Configure(ctx context.Context, req provider.Configu
 	resp.DataSourceData = client
 }
 
+func userAgent(name, providerVersion, tfVersion string) string {
+	ua := fmt.Sprintf("Terraform/%s (+https://www.terraform.io)", tfVersion)
```

Comment:
@yob will this cause issues for you monitoring if we change the format of the user agent? With this change it will look like: `Terraform/%s (+https://www.terraform.io) buildkite/v0.25.1` where I think it used to look like `Terraform/%s (+https://www.terraform.io) Terraform-Plugin-SDK/v2.10.0 buildkite/v0.25.1`

Reply:
totally fine, we don't have any processing of the user agent that depends on the precise format